src/dbnode/storage/mediator.go

```diff
@@ -123,19 +121,18 @@ func newMediator(database database, commitlog commitlog.CommitLog, opts Options)
 	cfm := newColdFlushManager(database, pm, opts)
 	d.databaseColdFlushManager = cfm
 
-	d.databaseRepairer = newNoopDatabaseRepairer()
-	if opts.RepairEnabled() {
-		d.databaseRepairer, err = newDatabaseRepairer(database, opts)
-		if err != nil {
-			return nil, err
-		}
-	}
-
 	d.databaseTickManager = newTickManager(database, opts)
 	d.databaseBootstrapManager = newBootstrapManager(database, d, opts)
 	return d, nil
 }
 
+func (m *mediator) RegisterBackgroundProcess(process BackgroundProcess) {
+	m.Lock()
+	defer m.Unlock()
```

Comment:
nit; Probably  better to just do this without defer after the append

Reply:
ok